src/Servers/IIS/AspNetCoreModuleV2/CommonLib/RedirectionOutput.cpp

```diff
@@ -77,7 +77,18 @@ void FileRedirectionOutput::Append(const std::wstring& text)
 {
     if (m_file.is_open())
     {
-        const auto multiByte = to_multi_byte_string(text, CP_UTF8);
+        auto multiByte = to_multi_byte_string(text, CP_UTF8);
+
+        // Writing \r\n to an ostream will cause two new lines to be written rather
```

Comment:
Will \n will be written without \r in the final file output?

Also, I don't think ostream was the issue, iirc it was a Windows API we were calling the duplicated the newline

Reply:
Hm, when I was testing it, I called ostream << "test\r\n" and I see two new lines in the file.

Reply:
Isn't this ostream the fake file that is then redirected? I could be wrong, it has been a long time since I looked at this

Reply:
I don't believe so, the m_file points directly to the stdout file.